crates/nu-command/src/math/utils.rs

```diff
@@ -94,3 +94,86 @@ pub fn calculate(
         )),
     }
 }
+
+// Utility functions to reduce boiler plate in the the
+// trigonometric math commands
+pub fn trig_value<F>(value: Value, use_degrees: Option<bool>, f: F) -> Value
```

Comment:
From this function name without doc comment it is unclear where the angle conversion is applied. Before the `F` like required for `sin`/`cos`/`tan` or after for the inverse/arcus functions like `arctan` also defined on the whole real numbers.

`value` as a name component strikes me as a bit superfluous thanks to the parameters and output types. Maybe a verb like `apply` or `inverse` is more fitting